pkg/rpc/server/server.go

```diff
@@ -1281,11 +1281,12 @@ func (s *Server) invokescript(reqParams request.Params) (interface{}, *response.
 
 	tx := &transaction.Transaction{}
 	if len(reqParams) > 1 {
-		signers, _, err := reqParams[1].GetSignersWithWitnesses()
+		signers, witnesses, err := reqParams[1].GetSignersWithWitnesses()
```

Comment:
This implies they'd be compete correct witnesses IIRC (see `invokefunction` notice in `rpc.md` also), I'm not sure C# ever returns not-really-signed transactions.

Reply:
> This implies they'd be complete correct witnesses IIRC

That's not necessary, you may provide dummy witnesses or partially filled witnesses as it is done for other `invoke*` requests and `calculatenetworkfee`. The only thing why we fill witnesses here is to provide a proper script container to VM.

> I'm not sure C# ever returns not-really-signed transactions

That's true, C# returns either a correctly signed transaction or an exception. But it's not a problem for Go node, because we never return transaction from `invoke*` calls and this behaviour is documented.

As a result, I still consider this commit should be applied because C# node parses both signers and witnesses and provide them to script container during `invokescript` handling (see [link](https://github.com/neo-project/neo-modules/blob/master/src/RpcServer/RpcServer.SmartContract.cs#L160)). We should match this behaviour to avoid related bugs, although witnesses are not included into `System.Runtime.GetScriptContainer` result and are not reachable from the transaction script.